app_check/src/ios/app_check_ios.mm

```diff
@@ -91,13 +92,60 @@ - (id)initWithProviderFactory:(firebase::app_check::AppCheckProviderFactory* _No
 
 @end
 
+@implementation AppCheckNotificationCenterWrapper
+
+// A collection of NSValues containing AppCheckListener*
```

Comment:
Should this be in the header?

Reply:
I'm a bit fuzzy on objective-C, but my understanding is that by defining NSMutableArray* listeners_ in the implementation block it is a private variable, whereas if it was in the interface it would be public. And it seems like an implementation detail that should be private.